### DISCUSSION TITLE
Déplacements entre les sections

### DISCUSSION THREAD

**Discussion by Nahrion** [vkbo/novelWriter#1702] [Q&A]
La question va paraitre un peu basic 
Je n'arrive pas à accéder aux fiches personnages. J'ai crée 2 personnages, mais je veux ajouter des informations dans leur profil. Le double clic, le clic, le bouton de droite... il ne se passe rien
Merci de votre aide

**Comment by vkbo** [comment]
There are no character sheets. See the docs: https://novelwriter.io/docs/project_references.html
